IM_calculation/IM/intensity_measures.py

```diff
@@ -2,8 +2,10 @@
 
 from IM_calculation.IM.rspectra_calculations import rspectra as rspectra
 from qcore import timeseries
+from IM_calculation.IM.Burks_Baker_2013_elastic_inelastic import Bilinear_Newmark_withTH
 
 DELTA_T = 0.005
+G = 981.0
```

Comment:
G is defined in im_calculation.py 

AI currently passed G in with the function. So would be good to bring these both inline with each other.

I don't have a preference as long as G isn't declared more than necessary (and ideally commented with what units it is in)

Reply:
Done. Kept in intensity_measures to avoid cross importing.